### PR TITLE
Added V support via v-analyzer.

### PR DESCRIPTION
**Pull request by PerilousBooklet**

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -809,6 +809,18 @@ lspconfig.vimls = add_lsp {
...
 }
 
+---# V
+--- __Status__: Works
+--- __Site__: https://github.com/vlang/v-analyzer
+--- __Installation__: https://github.com/vlang/v-analyzer?tab=readme-ov-file#installation
+lspconfig.v_analyzer = add_lsp {
+  name = "v_analyzer",
+  language = "v",
+  file_patterns = { "%.vv?$", "%.vsh$" },
```
Comment (Guldoman) [inline config.lua:819]: Should these patterns be added to `vls` too?

Comment (PerilousBooklet) [reply config.lua:819]: I think so, but I just read that [vls](https://github.com/vlang/vls) is no longer officially maintained as the official V lsp server, so I think I could update the description.
What do you think?

Comment (Guldoman) [reply config.lua:819]: Yep, go for it.

Comment (PerilousBooklet) [reply config.lua:819]: Ok, I'm on it.

**Pull request merged**